python/src/Sample_doc.i.in

```diff
@@ -856,7 +856,7 @@ n : int
 
 Parameters
 ----------
-indices : int or sequence of int, :math:`0 \leq i < n`
+indices : int, sequence of int, :math:`0 \leq i < n` or sequence of str
```

Comment:
Mention whats append if there are duplicate labels either in the sample description or in the filter description.

Reply:
ok